File: app/src/java/java/com/longtoast/bilbil_api/dto/ChatRoomListUpdateDTO.java
package com.longtoast.bilbil_api.dto;

import lombok.Data;
import lombok.Builder;
import java.time.LocalDateTime;

@Data
@Builder
public class ChatRoomListUpdateDTO {
    private Integer roomId;
    private String lastMessageContent;
    private LocalDateTime lastMessageTime;
    // private Integer partnerId; // 필요하다면 추가
}

File: app/src/main/java/com/longtoast/bilbil/dto/ChatRoomListUpdateDTO.kt
// com.longtoast.bilbil.dto.ChatRoomListUpdateDTO.kt
package com.longtoast.bilbil.dto

// 💡 [핵심 추가] 목록 업데이트 시 필요한 최소 정보 DTO
data class ChatRoomListUpdateDTO(
    val roomId: Int?,
    val partnerId: Int?, // 상대방 ID (정렬 및 UI 업데이트에 유용할 수 있음)
    val lastMessageContent: String?,
    val lastMessageTime: String?,
    val unreadCount: Int? = null
)